implementations/rust/ockam/ockam_transport_udp/Cargo.toml

```diff
@@ -15,7 +15,6 @@ edition = "2021"
 homepage = "https://github.com/ockam-network/ockam"
 keywords = ["ockam", "crypto", "network", "networking", "udp"]
 license = "Apache-2.0"
-publish = false
```

Comment:
Publish should be true for it to be releasable

Reply:
thanks @metaclips